GNUmakefile

```diff
@@ -138,24 +138,26 @@ test_targets += fvtest/porttest/aixbaddep
 endif
 endif
 
+ifeq (1, $(OMR_RAS_TDF_TRACE))
 # RAS Libraries
 main_targets += omrtrace
+# RAS Tests
+test_targets += fvtest/rastest
+endif
 
 # OMR Startup
 main_targets += omr omr/startup
 
-# RAS Tests
-test_targets += fvtest/rastest
-
 # OMR Example Targets
 ifeq (1,$(OMR_EXAMPLE))
 test_targets += example
+main_targets += omrtrace
```

Comment:
why is omrtrace getting adding multiple times?  It is here and above in the OMR_RAD_TDF_TRACE above.

Reply:
The `example` target links against omrtrace, and there exists no checks to ensure it has been enabled. This would prevent the errors that may arise when `OMR_RAS_TDF_TRACE` is disabled. Adding that simply helps with preventing build errors from being unable to find `omrtrace.a`. It may not be necessary to have this as a part of this PR, as it does not completely solve all the potential issues that may arise when omrtrace is disabled. There are more dependency-related issues that we could solve through assertions (through CMake only!) as well as more `#ifdef` guards across multiple components that use the trace engine.